src/GraphQL/Validation/ValidationContext.cs

```diff
@@ -470,25 +470,26 @@ public async ValueTask<Variables> GetVariableValuesAsync(IVariableVisitor? visit
 
                 var fields = inputType.Fields.List;
                 var fieldAsts = objValue.Fields;
-                if (fieldAsts == null)
-                    return null;
```

Comment:
These lines incorrectly skipped validation of object fields with no properties.

Reply:
😮   🤦‍♂️